config.py

```diff
@@ -32,7 +32,7 @@ class Config(object):
     The base configuration option. Contains the defaults.
     """
 
-    DEBUG = False
+    DEBUG = True
```

Comment:
By default DEBUG should be False, right? If someone wants to use DEBUG, she/he can use the DevelopmentConfig.